meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher.service

```diff
@@ -2,11 +2,11 @@
 Description=Resin init flasher service
 Requires=\
     mnt-boot.mount \
-    mnt-conf.mount
+    mnt-boot-config.json.mount
 Wants=resin-device-register.service
 After=\
     mnt-boot.mount \
```

Comment:
I think this can be removed as mnt-boot-config.json.mount already has an After for it